### PR TITLE
Poc 115

### PR DESCRIPTION
**Pull request by hiqedme**
family testing PR for release

### CODE REVIEW

File: src/app/shared/report-filters/report-filters.component.ts [hunk 1]
```diff
@@ -523,4 +614,28 @@ export class ReportFiltersComponent
     console.log('Location Type Change', $event);
```
Comment (kipchubett) [inline src/app/shared/report-filters/report-filters.component.ts]: Remove the console.log()

File: src/app/hiv-care-lib/family-testing/family-testing-base.component.ts [hunk 2]
```diff
@@ -26,22 +30,30 @@ export class FamilyTestingBaseComponent implements OnInit {
   public isEligible = '';
   public childStatus = '';
   public elicitedClients = '';
+  public hasLoadedAll = false;
+  public limit = 2000;
```
Comment (kipchubett) [inline src/app/hiv-care-lib/family-testing/family-testing-base.component.ts]: Why limit at `200`? does this implement pagination?

**Review by kipchubett — CHANGES_REQUESTED** [review]
@hiqedme 
Please resolve the conflicts and squash your commits

File: src/app/hiv-care-lib/family-testing/family-testing-base.component.ts (hunk 2 above)
Comment (hiqedme) [reply src/app/hiv-care-lib/family-testing/family-testing-base.component.ts]: we'll resolve this one after tests as we aren't sure about why that particular value was used

**Pull request closed without merging**